app/views/new_event_mailer/notify_volunteer.html.slim

```diff
@@ -0,0 +1,8 @@
+We have just announced another event and need volunteers like you!
+
+h2 = @event.event_name
+h4 = @event.location
+p = @event.description
+
+To sign up for this event please visit the event page
+  link to "event page" @event
```

Comment:
extra space in front. and should be `= link_to "event page", event_url(@event)` 
Take note of the path, we are using `*_url` because emails are already outside of our web app, and therefore Rails cannot provide you a relative path. e.g `/events/1` will go nowhere